setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup, find_packages
 
-requirements = ["mazer", "packaging", "pulpcore-plugin~=0.1rc3"]
+requirements = ["galaxy_importer==0.1.0", "mazer", "packaging", "pulpcore-plugin~=0.1rc3"]
```

Comment:
FYI pinning to specific version for now, while adding functionality to galaxy-importer